astrolabe/cli.py

```diff
@@ -613,8 +615,11 @@ def delete_test_cluster(ctx, spec_test_file, workload_file, org_id, project_name
     if project:
         try:
             ctx.obj.client.groups[project.id].clusters[cluster_name].delete()
-        except AtlasApiBaseError:
-            pass
+            print(f"{msg} done.")
+        except AtlasApiBaseError as e:
+            pprint(e)
```

Comment:
Should we re-raise this so the task fails if deleting the cluster fails? Is there something else we can do to make it more obvious that we've left an expensive resource hanging around?

Reply:
This occurs during a tear-down tastk so failing wouldn't fail the build.

Reply:
Good point. We could use `post_error_fails_task: true` to make post functions fail the task.

Reply:
Created [DRIVERS-2729](https://jira.mongodb.org/browse/DRIVERS-2729) to track that suggestion.